NOTES.WIN

```diff
@@ -35,24 +35,50 @@
  In addition to the requirements and instructions listed in INSTALL,
  these are required as well:
 
- - Perl. We recommend ActiveState Perl, available from
-   https://www.activestate.com/ActivePerl. Another viable alternative
-   appears to be Strawberry Perl, http://strawberryperl.com.
-   You also need the perl module Text::Template, available on CPAN.
+ - Perl.
+   We recommend Strawberry Perl, available from http://strawberryperl.com.
+   An alternative is ActiveState Perl, https://www.activestate.com/ActivePerl.
+   You may also need the perl modules Text::Template and Win32::Console,
```

Comment:
Actually, to make it clear that this is for ActivePerl, so may I suggest joining the sentences, i.e. like this?

```
 - Perl.
   We recommend Strawberry Perl, available from http://strawberryperl.com.
   An alternative is ActiveState Perl, https://www.activestate.com/ActivePerl,
   for which you may need the perl modules Text::Template and Win32::Console,
   available on CPAN or https://platform.activestate.com/ActiveState.
```

Reply:
The reference to `Text::Template` was already there, and I suspect that at least that module is needed (but already included) with Strawberry Perl. 
CPAN is likely not helpful for ActivePerl.

Reply:
I'm frankly not sure why Text::Template is needed, as we include it in our tarballs...  and Win32::Console is currently only needed for ActivePerl.  The way this is currently phrased, it sounds like both are needed for Strawberry as well, and I see no reason why we should send our users on that goose chase unnecessarily.

Reply:
I've just changed this to:
```
An alternative is ActiveState Perl, https://www.activestate.com/ActivePerl
for which you may need to explicitly select the Perl module Win32/Console.pm
available via https://platform.activestate.com/ActiveState.
```

Reply:
Yes!  Really good.  Thanks for your patience

Reply:
And thank you for the swift feedback & approval.